libs/fgdbg/include/fgdbg/Resource.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FGDBG_RESOURCE_H
+#define FGDBG_RESOURCE_H
+
+#include <string>
+
+namespace filament::fgdbg {
```

Comment:
Resource.h and Pass.h could probably be consolidated to a single header, unless you're planning on adding a lot more to them.

Reply:
I think that's a good idea! What do we usually call the enclosing .h file `models.h`?